packages/queries/src/query.ts

```diff
@@ -100,10 +100,41 @@ export class QueryBuilder<
         );
     }
 
+    /**
+     * Returns all rows that match the query filters.
+     *
+     * Does not override the limit set by `.limit()`.
+     *
+     * Alias for `.many()`.
```

Comment:
We could also add this particular line of comment for the other two, so "Alias for `.maybe()`" and "Alias for `one()`"